gremlin-go/driver/bytecode_test.go

```diff
@@ -117,4 +117,20 @@ func TestBytecode(t *testing.T) {
 			assert.Equal(t, testValue, bc.bindings[testKey])
 		})
 	})
+
+	t.Run("traversal", func(t *testing.T) {
+		bc := bytecode{}
+		traversal := Traversal{}
+		traversal.graph = nil
+		traversal.bytecode = &bytecode{}
+		traversalBytecode, err := bc.convertArgument(traversal)
+		assert.Nil(t, err)
+		assert.Equal(t, traversal.bytecode, traversalBytecode)
+
+		traversal.graph = &Graph{}
+		traversal.bytecode = &bytecode{}
+		traversalBytecode, err = bc.convertArgument(traversal)
+		assert.NotNil(t, err)
```

Comment:
Would it be better to test the failure case as a seperate test or will the combination suffice?

Reply:
Good point - will separate.